README.md

```diff
@@ -34,67 +34,56 @@ For a more detailed overview of the repository, please see the documents on the
 
 ## Getting Started
 
+We recommend [conda](https://docs.conda.io/projects/conda/en/latest/glossary.html?highlight=environment#conda-environment) for environment management, and [vscode](https://code.visualstudio.com/) for development. To install the recommenders package and run an example notebook:
+
+```bash
+# Create and activate a new conda environment
+conda create -n <environment_name> python=3.9
+conda activate <environment_name>
+
+# Install the recommenders package with examples
+pip install recommenders[examples]
+
+# create a Jupyter kernel
+python -m ipykernel install --user --name <environment_name> --display-name <kernel_name>
+
+# Clone this repo within vscode or using command:
+git clone https://github.com/microsoft/recommenders.git
+
+# Within vscode:
+#   1. Open a notebook, e.g., examples/00_quick_start/sar_movielens.ipynb;  
+#   2. Select Jupyter kernel <kernel_name>;
+#   3. Run the notebook.
+```
+
+For more information about setup including extras, as well as configurations for GPU, Spark and Docker container, see the [setup guide](SETUP.md).
+<!--
+In addition to the core package, several extras are provided, including:
++ `[examples]`: Needed for running examples.
++ `[gpu]`: Needed for running GPU models.
++ `[spark]`: Needed for running Spark models.
++ `[dev]`: Needed for development for the repo.
++ `[all]`: `[examples]`|`[gpu]`|`[spark]`|`[dev]`
++ `[experimental]`: Models that are not throughly tested and/or may require additional steps in installation).
++ `[nni]`: Needed for running models integrated with [NNI](https://nni.readthedocs.io/en/stable/).
+-->
```

Comment:
This part is very interesting. Thinking as a user, in a quick view, I could understand that there are multiple flavours. I think if we add this we skip the user one step (that I would make) which is to install recommenders[examples], run one of the GPU examples, get an error, then go back, realize that I had to do recommenders[gpu,examples], and then run it. What do you think?